modules/virt-configuring-secondary-network-vm-live-migration.adoc

```diff
@@ -6,57 +6,55 @@
 [id="virt-configuring-secondary-network-vm-live-migration_{context}"]
 = Configuring a dedicated secondary network for virtual machine live migration
 
-To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition for the `openshift-cnv` namespace by using the CLI. Then, add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
+To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition (NAD) by using the CLI. Then, you add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
 
 .Prerequisites
 
 * You installed the OpenShift CLI (`oc`).
 * You logged in to the cluster as a user with the `cluster-admin` role.
-* The Multus Container Network Interface (CNI) plugin is installed on the cluster.
-* Every node on the cluster has at least two Network Interface Cards (NICs), and the NICs to be used for live migration are connected to the same VLAN.
-* The virtual machine (VM) is running with the `LiveMigrate` eviction strategy.
```

Comment:
Not sure about this one - Is the eviction strategy no longer a prerequisite

Reply:
Eviction strategy is not a prerequisite for creating a migration network. It's a prerequisite for migrating the VM. That's why I'm removing it from this procedure.